### PR TITLE
Fixed CLI bug

### PR DESCRIPTION
**Pull request by pinebit**
Closes https://app.shortcut.com/chainlinklabs/story/50270

NOTE: `admin users update` command is replaced with `admin users chrole`. See CHANGELOG.

### CODE REVIEW

**Review by jmank88 — DISMISSED** [review]

File: core/cmd/admin_commands.go [hunk 1]
```diff
@@ -130,22 +130,14 @@ func (cli *Client) CreateUser(c *cli.Context) (err error) {
 
 // EditUser can change a user's email, password, and role
```
Comment (bolekk) [inline core/cmd/admin_commands.go:131]: update comment

File: core/sessions/orm.go [hunk 2]
```diff
@@ -281,17 +281,6 @@ func (o *orm) UpdateUser(email, newEmail, newPassword, newRole string) (User, er
...
-			pwd, err := ValidateAndHashPassword(newPassword)
-			if err != nil {
-				return err
-			}
-			userToEdit.HashedPassword = pwd
-			// Password changed, purge associated sessions at final step
-			purgeSessions = true
-		}
-
 		// Patch validated role is newRole is specified
```
Comment (bolekk) [inline core/sessions/orm.go:284]: "is" -> "if"

File: core/chains/evm/log/mocks/orm.go [hunk 3]
```diff
@@ -15,20 +15,20 @@ type ORM struct {
 	mock.Mock
 }
 
-// CreateBroadcast provides a mock function with given fields: blockHash, blockNumber, txIndex, logIndex, jobID, qopts
```
Comment (bolekk) [inline core/chains/evm/log/mocks/orm.go:18]: This file shouldn't be a part of this PR, right?

Comment (pinebit) [reply core/chains/evm/log/mocks/orm.go:18]: We usually commit the mocks, and I had to regenerate mocks due to changed ORM interface.

**Review by essamhassan — APPROVED** [review]
LGTM

**Review by jmank88 — APPROVED** [review]

**Pull request merged**